blocked-edges/4.9.yaml

```diff
@@ -0,0 +1,3 @@
+to: 4\.9\..*
```

Comment:
[`to` doesn't support regular expressions][1] (only [`from` does][2]), so you'll want to iterate over all 4.9.z and create a copy of this that blocks updates from 4.8.  Amazingly, this is our first to-4.9 block, so we don't need to worry about merging with existing to-4.9 issues.  You can just iterate over all the 4.9.z in `fast-4.9`.

[1]: https://github.com/openshift/cincinnati-graph-data/blame/d8c05513732b51a2a49735497609bcd0c945c1a2/README.md#L102
[2]: https://github.com/openshift/cincinnati-graph-data/blame/d8c05513732b51a2a49735497609bcd0c945c1a2/README.md#L103